doc/classes/@GlobalScope.xml

```diff
@@ -1459,148 +1497,148 @@
 		<constant name="KEY_KP_9" value="16777359" enum="Key">
 			Number 9 on the numeric keypad.
 		</constant>
-		<constant name="KEY_SUPER_L" value="16777260" enum="Key">
+		<constant name="KEY_SUPER_L" value="16777279" enum="Key">
```

Comment:
`SUPER_L = SPECIAL | 0x40 = int(16777216) + int(64) = 16777280`
You can see what you need to change in https://github.com/godotengine/godot/runs/6876044106?check_suite_focus=true.

Reply:
Ohh I forgot to rebuild my binary, that's why I got the wrong XML after running the binary with `--doctool` 
today I learned something new again :D